Tella/Application/TellaApp.swift

```diff
@@ -26,6 +27,10 @@ struct TellaApp: App {
                     if value {
                         self.saveData(lockApptype: .finishBackgroundTasks)
                     }
+                }.onAppear {
+                    gDriveAuthViewModel.restorePreviousSignIn()
```

Comment:
I added this on the onAppear method to follow the convention from [google doc](https://developers.google.com/identity/sign-in/ios/sign-in#2_attempt_to_restore_the_users_sign-in_state). The idea is to refresh the token every time the user opens the app. 

We could add a condition to only restorePreviousSignIn if the user has a g-drive server connection so we reduce the amount of calls. What do you think about this?

Reply:
`We could add a condition to only restorePreviousSignIn if the user has a g-drive server connection` in this case we need to login ?

Reply:
no, but we will need a token for submitting reports to drive. the idea of restorePreviousSignIn is to avoid making the user to re-authenticate and don't let the token expire